pkg/vault/test/foundry/VaultSwap.t.sol

```diff
@@ -546,13 +544,17 @@ contract VaultSwapTest is BaseVaultTest {
 
         (uint256 fee, uint256 protocolFee) = testFunc();
 
+        // Pool fees are taken on the calculated end
         if (kind == SwapKind.EXACT_OUT) {
+            // Fees applied to token in (user pays more usdc)
             usdcFee = fee;
-            usdcProtocolFee = protocolFee;
         } else {
+            // Fees applied to token out (user gets less dai).
             daiFee = fee;
-            daiProtocolFee = protocolFee;
         }
```

Comment:
Just to check if I understood this part: To the Vault, fees are always taken in terms of token in amounts, but it's transparent to the user. So, the user feels like it's paying fees with the calculated end. We can calculate the usdc and dai fees like this because the pool is linear and the rates are 1, so the token in fee and the token calculated fee would be the same. Is that it?